src/System.Numerics.Tensors/System/Numerics/DenseTensor.cs

```diff
@@ -98,6 +98,29 @@ public override void SetValue(int index, T value)
             Buffer.Span[index] = value;
         }
 
+        protected override void CopyTo(T[] array, int arrayIndex)
+        {
+            Buffer.Span.CopyTo(array.AsSpan().Slice(arrayIndex));
+        }
+
+        protected override int IndexOf(T item)
+        {
+            // TODO: use Span.IndexOf when/if it removes the struct type constraint
```

Comment:
This should be complete soon: https://github.com/dotnet/corefx/pull/24911

Reply:
It seems I still can't use the method since it requires `where T : IEquatable<T>`.  We don't have that constraint either.